src/grammar.lalrpop

```diff
@@ -311,6 +311,50 @@ ClauseGuard4: UntypedClauseGuard = {
 }
 
 ClauseGuard5: UntypedClauseGuard = {
+    <s:@L> <left:ClauseGuard5> ">." <right:ClauseGuard6> <e:@L> => ClauseGuard::GtFloat {
```

Comment:
To match how the parsing of regular expressions works for these operators we can put all of these new operators in the `ClauseGuard4` block, we don't need to create a new block for each one.


```rust
ClauseGuard4: UntypedClauseGuard = {
    <s:@L> <left:ClauseGuard4> ">" <right:ClauseGuard5> <e:@L> => ClauseGuard::GtInt {
        location: location(s, e),
        typ: (),
        left: Box::new(left),
        right: Box::new(right),
    },

    <s:@L> <left:ClauseGuard4> ">." <right:ClauseGuard5> <e:@L> => ClauseGuard::GtFloat {
        location: location(s, e),
        typ: (),
        left: Box::new(left),
        right: Box::new(right),
    },

  // And so on...

    ClauseGuard5 => <>,
}
```